source/Calamari.Tests/KubernetesFixtures/KubernetesContextScriptWrapperLiveFixtureAks.cs

```diff
@@ -184,7 +184,7 @@ public void DiscoverKubernetesClusterWithAzureServicePrincipalAccount(bool setHe
                 { "awsAssumeRole", bool.FalseString }
             };
 
-            if (scope.HealthCheckContainer is not null)
+            if (scope.HealthCheckContainer != null)
```

Comment:
Turns out this was the only issue keeping Calamari.Tests from being set to C# version 7.3 so I've just updated it to use the older style of null check.

Reply:
Rider doesn't like when you change compiler versions for projects as it will show errors where there are none until you delete the bin and obj folders for that project and rebuild.